src/app/index.html

```diff
@@ -11,4 +11,7 @@
   <li>
     <a skyAppLink="/visual/toggle-switch">Toggle switch</a>
   </li>
+  <li>
+    <a href="#" skyAppLink="/visual/character-count">Character Count</a>
```

Comment:
You can remove the `href` attribute when using the `skyAppLink` directive. Also, do you mind placing this menu item in alphabetical order?